packages/amplication-notifications-service/src/providers/secrets/base/secretsManager.service.base.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { ConfigService } from '@nestjs/config';
+import { mock } from 'jest-mock-extended';
+import { SecretsManagerServiceBase } from './secretsManager.service.base';
+
+describe('Testing the secrets manager base class', () => {
+  const SECRET_KEY = 'SECRET_KEY';
+  const SECRET_VALUE = 'SECRET_VALUE';
+  const configService = mock<ConfigService>();
+  const secretsManagerServiceBase = new SecretsManagerServiceBase(
+    configService
+  );
+  beforeEach(() => {
+    configService.get.mockClear();
+  });
+  it('should return value from env', async () => {
```

Comment:
I'd rephrase this to:
`should get value from secret manager`

Reply:
this is an auto-generated file by Amplicaton